docs/setup/install/install-devtron-helm-3.md

```diff
@@ -63,6 +63,11 @@ kubectl -n devtroncd get installers installer-devtron -o jsonpath='{.status.sync
 ```
 
 The install commands initiates Devtron-operator which spins up all the Devtron micro-services one by one in about 20 mins. You can use the above command to check the status of the installation if the installation is still in progress, it will print `Downloaded`. When the installation is complete, it prints `Applied`.
+During devtron installation process, you can check logs of installer. To check logs of installer, run the following command :
```

Comment:
Replace the content, as mentioned above.

Reply:
done